### PR TITLE
chore: add aspects plugin

### PR DESCRIPTION
**Pull request by Ian2012**
This PR adds the aspects plugin to the list of contrib plugins.

Aspects support multiple openedx versions, so I'm adding that here too

### CODE REVIEW

**Review by regisb — CHANGES_REQUESTED** [review]

File: contrib/redwood/plugins.yml [hunk 1]
```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: aspects
+  src: tutor-contrib-aspects==1.2.0
```
[the remark this answers is not in the document]
Comment (Ian2012) [reply contrib/redwood/plugins.yml:29]: We will have a v1.2.1 release soon and then drop support for python3.8 in multiple places. @bmtcril what do you think the version constraint should be? `<3.0`

[the remark this answers is not in the document]
Comment (Ian2012) [reply contrib/redwood/plugins.yml:29]: I can also use <2.0.0 while 2.0.0 is confirmed

[the remark this answers is not in the document]
Comment (bmtcril) [reply contrib/redwood/plugins.yml]: <2 should be good for now. 2.x will be the breaking version.

[the remark this answers is not in the document]
Comment (Ian2012) [reply contrib/redwood/plugins.yml]: @regisb should I also add a lower bound? `>1.0.0`?

**Review by DawoudSheraz — APPROVED** [review]

File: contrib/redwood/plugins.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (regisb) [reply contrib/redwood/plugins.yml]: I have no idea, it's really up to you.

File: contrib/olive/plugins.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+- name: aspects
+  src: tutor-contrib-aspects<2.0.0
+  url: https://github.com/openedx/tutor-contrib-aspects
+  author: Brian Mesick <bmesick@axim.org>
+  maintainer: Brian Messick <bmesick@axim.org>
```
Comment (regisb) [inline contrib/olive/plugins.yml:5]: Messick -> Mesick
(same below)

**Pull request merged**